src/GraphQL.ApiTests/GraphQL.approved.txt

```diff
@@ -592,6 +592,21 @@ namespace GraphQL.DataLoader
 }
 namespace GraphQL.Execution
 {
+    public enum ArgumentSource
+    {
+        FieldDefault = 0,
+        Literal = 1,
+        Variable = 2,
+        VariableDefault = 3,
+    }
+    public readonly struct ArgumentValue
+    {
+        public ArgumentValue(object value, GraphQL.Execution.ArgumentSource source) { }
+        public GraphQL.Execution.ArgumentSource Source { get; }
+        public object Value { get; }
+        public static GraphQL.Execution.ArgumentValue NullLiteral { get; }
+        public static GraphQL.Execution.ArgumentValue NullVariable { get; }
```

Comment:
Keep in mind that these properties are not used within GraphQL, and `default` can be used for any tests that need a `null` value and don't care about `Source`.  I probably didn't think of that when I changed the tests previously.  I'd remove these properties and change `ArgumentValue.NullLiteral` to `default`.  But I'm ok if you want to leave them.

Reply:
> and don't care about Source

This can be a source of errors. I want to leave them because such properties may be useful.